.github/workflows/py-checks.yml

```diff
@@ -1,15 +1,20 @@
 name: Code Checks
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'  # Runs every Sunday at midnight to clear the cache
```

Comment:
Could you elaborate on how scheduling this is clearing the cache?  
By my understanding the cache is cleared based on key invalidation, which only really seems to happen when the hash for the requirements file changes.

Reply:
Actually, lets just remove the schedule all together.  Cache is cleared automatically by some server-side, LRU-like process.